dtfs-central-api/src/v1/controllers/utilisation-report-service/fee-record-correction/put-fee-record-correction-transient-form-data.controller/helpers/parse-record-correction-transient-form-values.ts

```diff
@@ -0,0 +1,22 @@
+import { getMonetaryValueAsNumber, isCurrencyValid, RecordCorrectionFormValues, RecordCorrectionTransientFormData } from '@ukef/dtfs2-common';
+
+/**
+ * Parses validated record correction transient form values.
+ * @param formValues - The validated form data containing record correction values.
+ * @returns A RecordCorrectionTransientFormData object with parsed monetary values and form fields.
+ */
+export const parseValidatedRecordCorrectionTransientFormValues = (formValues: RecordCorrectionFormValues): RecordCorrectionTransientFormData => {
+  const utilisation = formValues.utilisation ? getMonetaryValueAsNumber(formValues.utilisation) : undefined;
+  const reportedCurrency = isCurrencyValid(formValues.reportedCurrency) ? formValues.reportedCurrency : undefined;
+  const reportedFee = formValues.reportedFee ? getMonetaryValueAsNumber(formValues.reportedFee) : undefined;
+  const additionalComments = formValues.additionalComments?.trim();
```

Comment:
This might be influenced by my Friday brain, but I think this could do with some new lines in between some of them